blog/docs/articles/eventing-kafka-strimzi-tls.md

```diff
@@ -0,0 +1,323 @@
+# How To: process events coming from an Apache Kafka cluster managed by Strimzi operator with TLS authentication and encryption
+
+**Date: 2023-05-18**
+
+***Author: [Pierangelo Di Pilato](https://twitter.com/pierDipi), Knative Eventing Working Group
+Lead, Software Engineer @ Red Hat***
+
+**Introduction**
+
+Apache Kafka has gained immense popularity as a distributed streaming platform, allowing developers
+to build scalable and fault-tolerant event-driven applications. Strimzi, an open-source project,
+simplifies the management of Apache Kafka on Kubernetes by providing operators for Kafka and
+ZooKeeper. In this tutorial, we will explore how to process events from an Apache Kafka cluster
+managed by the Strimzi operator using Knative Eventing KafkaSource.
+
+**Prerequisites**
+
+To follow this tutorial, you should have the following prerequisites in place:
+
+1. A Kubernetes cluster.
+2. Strimzi operator installed, see [deploy Strimzi](https://strimzi.io/quickstarts/).
+3. Knative Eventing installed on the Kubernetes cluster, see [Install Knative Eventing](https://knative.dev/docs/install/yaml-install/eventing/install-eventing-with-yaml/#install-knative-eventing).
+4. Knative Eventing Kafka Source installed on the Kubernetes cluster, see [Install the KafkaSource controller](https://knative.dev/docs/eventing/sources/kafka-source/#install-the-kafkasource-controller).
+
+**Step 1: Create a Kafka cluster**
+
+```shell
+kubectl apply -f - <<EOF
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: my-kafka-cluster
+  namespace: kafka
+spec:
+  kafka:
+    replicas: 3
```

Comment:
since we don't specifiy a version here, should we add a note on the strimzi defaulting?